src/main/js/component/common/ConfigurationLabel.js

```diff
@@ -47,9 +48,7 @@ class ConfigurationLabel extends Component {
         return (
             <div className="d-inline-flex col-sm-4">
                 <h1>
-                    {fontAwesomeIcon &&
-                    <span className={iconClasses} />
-                    }
+                    <FontAwesome name={fontAwesomeIcon} fixedWidth />
```

Comment:
Do we want to wrap this in
`
{fontAwesomeIcon &&
...
} 
`
like we had before, so it will only display if the icon is set

Reply:
I was worried I was removing a feature with this removal. At the moment, is there any page we use that doesn't use fontawesome? At the moment, it's almost a requirement for each descriptor.

Reply:
If that is the case, then we should update the proptypes to make this property required